### PR TITLE
Ch travis intergration

### PR DESCRIPTION
**Pull request by sirghiny**
What this PR does:
- Sets up TravisCI integration
- Fixes failing tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,17 +15,12 @@ sudo: false
...
   - "2.7"
-  - "3.4"
   - "3.5"
 
 # Manually define here the combinations environment variables to test
 # https://github.com/travis-ci/travis-ci/issues/1519
 env:
   - TEST_MOBILE=True  DB=postgresql TRAVIS_NODE_VERSION="4"
-  - TEST_MOBILE=True  DB=sqlite     TRAVIS_NODE_VERSION="4"
-  - TEST_MOBILE=False DB=postgresql TRAVIS_NODE_VERSION="4"
```
Comment (egichuri) [inline .travis.yml]: lets also include `TEST_MOBILE=False` for postgres as one of the builds

File: .travis.yml [hunk 2]
```diff
@@ -43,13 +38,17 @@ install:
...
+  - pip install wger
+  - if [[ "$DB" = "postgresql" ]]; then wger create_settings --database-type postgresql; fi
+
 
 # Create test databases
 before_script:
   - if [[ "$DB" = "postgresq" ]]; then psql -c 'DROP DATABASE IF EXISTS test_wger;' -U postgres; fi
   - if [[ "$DB" = "postgresql" ]]; then psql -c 'CREATE DATABASE test_wger;' -U postgres; fi
+  - npm install -g gulp
+  - npm install --global gulp-cli
```
Comment (egichuri) [inline .travis.yml]: Is there a reason why this was added?

File: package.json [hunk 3]
```diff
@@ -25,6 +25,10 @@
     "eslint-plugin-jsx-a11y": "^2.1.0",
     "eslint-plugin-react": "^6.1.2",
     "gulp": "^3.9.1",
-    "gulp-eslint": "^3.0.1"
+    "gulp-eslint": "^3.0.1",
+    "redux-logger": "^3.0.6"
+  },
+  "dependencies": {
+    "bower": "^1.8.0"
```
Comment (egichuri) [inline package.json]: this too?

**Review by egichuri — CHANGES_REQUESTED** [review]

**Comment by egichuri** [comment]
let's change the base branch for this pr to `develop`

File: .travis.yml (hunk 2 above)
Comment (zmwas) [reply .travis.yml]: So that gulp lint can run. It can't run when installed from packages json.

File: package.json (hunk 3 above)
Comment (zmwas) [reply package.json]: Those are dependencies added when you install the app.

File: .travis.yml (hunk 1 above)
Comment (egichuri) [reply .travis.yml]: @sirghiny please change this before we merge

**Review by egichuri — APPROVED** [review]

**Pull request merged**